Comparison shopping engine/Scrapers/Selenium/RdeScraper.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Comparison_shopping_engine.Selenium;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Comparison_shopping_engine.Selenium
+{
+    class RdeScraper : AbstractSeleniumScraper
+    {
+        protected override void NavigateToNextPage(ChromeDriver driver)
+        {
+            string beforeUrl = driver.Url;
+            string[] currentUrl = driver.Url.Split('/');
+            var url = "https://rde.lt/search/lt/word/" + currentUrl[6] + "/page/" +
```

Comment:
var nuorodos = driver.FindElements(By.CssSelector(".pages"));
if (nuorodos.Count > 0){
var puslapiai = nuorodos.FindElements(By.TagName("a"));
}
var nextPage = puslapiai[puslapiai.Count-2];

Ar kartais netiktų toks kodas?

Reply:
Kodėl būtent .pages ? Nelabai supratau sitos dalies
Mano logika tuomet rastu tik 2 .pages, virsutine ir apatine pasirinkimų juostą, tai reiskia ju countas niekad nebus didesnis už 3. Iš to seką jog nepateksim i kitą puslapį. Nebent iškarto ieškotume su tagname "a", bet tada reiktu daryti kitokį if'ą manyciau.
Pagrinde kodėl rasiau toki ilgą koda, tai nes kito puslapio pasirinkimas, neturi aiskaus tagname ar class. O nuėjus iki paskutinio puslapio išviso dingsta tas pasirinkimas. Šiuo atveju manau ir "var nextPage = puslapiai[puslapiai.Count-2];" netiktu.
Gal galėtum labiau paaiškinti, jeigu kažko nesupratau?

Reply:
oj, pats suklydau, 
var puslapiai = nuorodos[0].FindElements(By.TagName("a"));
turėjo atrodyti sitaip
bet vistiek dar kartelį patikrinau, toks kodas neveiktų, nes paskutiniuose puslapiuose tos rodyklėlės nebėra kuri permestų į sekantį puslapį